Replication Project_Final.Rmd

```diff
@@ -0,0 +1,994 @@
+---
+title: "Replication Project"
+author: "Andrés Lajous and Leah Reisman"
+date: "March 11, 2015"
+output: html_document
+---
+
```

Comment:
I read through the diffs (https://github.com/soc504-s2015-princeton/replication/pull/5/files) and your changes all look good.  I think the code is a bit clearer now. :+1: